package.json

```diff
@@ -3,6 +3,12 @@
     "react-native-get-random-values": "1.4.0",
     "react-native-reanimated": "2.4.1",
     "react-native-safe-area-context": "3.2.0",
+    "react-native-screens": "2.9.0",
+    "react-native-svg": "9.13.6",
+    "react-native-webview": "11.6.5",
+    "@react-native-masked-view/masked-view": "0.2.6",
+    "@react-native-clipboard/clipboard": "1.9.0",
+    "react-native-gesture-handler": "2.2.0",
```

Comment:
The forked repository of this package has modifications that were required to fix an issue that was already solved in `trunk` but not released yet. Now that they include it in version `[2.3.1](https://github.com/software-mansion/react-native-gesture-handler/releases/tag/2.3.1)`, we could bump it to that version.

Reply:
I applied this change in https://github.com/wordpress-mobile/react-native-libraries-publisher/pull/6/commits/850a64e983c4d0d190ff4af308263934a796e818.